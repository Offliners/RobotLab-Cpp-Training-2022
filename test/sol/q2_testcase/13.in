578
NUIsANCE
expeL
iNteRSectiON
twIn
pride
MAsTErY
tuMor
gROUP
Bingo
rAiD
laCE
dElICATE
TUTOR
NINETEEN
TASK
TOOThpASTE
OuTStAnDIng
DonATe
bAsIc
STAnDpOinT
PROFESsoR
DragOnfly
GREENHOUSE
provincial
faLl
like
BlanK
CONTeNT
THiNK
MILLIgrAM
modernIzaTion
FOReCAST
snap
ELEGANT
thunder
TEmPLE
Dance
aSPiRation
CArEsS
PROBABLE
PRefAcE
riDdLe
ANt
wEb
chamber
AvEnue
but28E7
stAbility
BATHTUB
EVIl
guLF
eNabLe
baptIsm
sCArF
resist
GROOM
An2el
EXPeCT
eXTENSiVE
dEsiGNEr
SunSHine
fraMe
GrIEvE
US
GoODnESs
inhAbitanT
jUNk
QUARTER
SheEP
FEET
acTUAl
CArrOt
66P96a9
EIgHTY
saddLe
AccESsory
Flock
ecoNoMICaL
BRACE
MISSION
f1Gh9
cULT
SKirT
rebEl
ORDINARILY
resistance
GreED
pioneer
APPLAUSE
secretary
WIDEN
AGGrEGAtE
wheelcHAir
PolICY
tEACh
BRAVERy
CLUMsY
cOmFoRTABLy
disadvantage
TRAvel
Panic
ItCH
UNPreDICTABLE
HANDBOOk
humIliATe
A0C020te
clUster
nE9
DRAwbACK
wednesday
calL
lauNdRY
FLOAT
SHORE
COURTEOUS
hypocrisy
pineapple
bEHaVE
INSTRUcTION
pARK
ReStroom
ANALOgY
BECOmE
72O47
bankRUPT
Per
wAIt
PowdeR
DATA
tuMpiKe
DiStresS
cApture
BLOC
aSHAMed
interpreter
grEet
FaILURE
associate
MINiStER
sTAte
MARGINAL
rEpuBLiCaN
DIvErSIFy
compleTe
Out-oF-Date
HAIRSTYLE
Mortal
iNVISibLE
conTRarY
MAID
gAp
pedesTrian
make
baT
mAYoR
PersevEraNce
nightmaRe
TELEphONE
FeMAlE
de81r1
982356S8
repoRTer
DUty
irritable
Release
cURious
SHIft
misTAke
life
cliCk
liVE6
eNThusIasTIC
9O9S5645ioN
opErationaL
NicKName
DiFFeRentiAl
RAY
tRIAL
divide
uNit
HyPOCrITe
homeland
elSE
gOlden
aTtaInmenT
SoVEREIGN
PrOUd
STorAge
TEENAGER
park
RECORD
abIDe
sTeward
9065E0
sIng
HYPOTHEsiS
HUmblY
COINCIDENCE
AMUsE
CHILDLIKE
auToBiography
Oven
eThIc
TREAsURE
poLE
pAwn
sTAy
STIR
SoCIAlly
JUDGE
neglIgibLe
ChaNCe
swEEt
Nephew
draw
LOYaLTY
VISIBLe
iNdiffERenT
bARBeR
PotATo
AGEnt
pitcher
EARN
ADaptatION
COMPLICATE
dropout
PATTERN
muSEUM
bartenDeR
aMAZE
boOm
iRrEsPOnsIble
neglEct
liGhtninG
rEmAIn
rOLl
IMAGe
MELANChOLY
diamOnd
rEhEARsE
OWNERSHIP
69un719
mANNERS
STrAIN
university
LINGUIstICS
DEMOCRACY
sEaSon
hUmiLIate
6A434380
FEvER
eVolUtion
ENVIRONMENt
CROUCH
NewScaSteR
onCe
beef
DRAINAGE
Go
BEIJING
DEFiNIte
HABit
SCRatCH
fIniSh
STRAw
COnSciEntioUS
025215a5123
DentaL
TYPe
WHaTEVer
PLASTER
EXERcIse
cOCKROaCH
IdEntIFIcAtiON
iNVent
PORTER
JUmP
PENNY
LAP
TaBLOID
smooth
FUrTHER
surface
alley
select
hAppiness
NINETY
COMMuNICATION
INGrEdient
oBSTACLE
siZaBlE
PaInFully
ReSUlTaNT
Cause
BaRbARIaN
howl
AMaZinG
s5Bd1E
MoNOTOny
ecONoMicS
Brown
ELECT
NAp
INTerchangEablE
privaTeLy
touchstone
qUArrEL
ELEcTric
MAGnET
array
LIBerAL
HONEY
cOMPAnY
twinklE
S91F4
politicaL
MANY
PrinCipAlly
FORFEIT
CAndLE
can
someday
RELUCTANTlY
country
GENERATION
REVOLUTION
INTELLIgENT
CrOwdEd
HOLIDAY
DISManTle
outDoOr
IntERESt
ATTRAcTIoN
reSErvATion
clutch
INDOORS
dOwnStAirs
visiTor
SUNLIGHT
eSseNTIAL
FANcY
BOxInG
sPonsor
Close
ReLATIVELY
heADing
SIMILArLy
iTALIAN
NAG
ALMOST
ENGiNEeR
dEfeNSive
BIOCHEMiSTrY
responsibIlITy
LINgERing
dePiCt
apparent
aCtIViTY
RESOLuTIOn
ability
EMiGRANT
evEry
Humbly
kiNG-SizE
iNdoOr
PILe
rEcYCLe
conSUMptiOn
Optimism
trAnS70rmAtION
sLeEp
meNu
CoMpETITOr
ARM
enough
28N36H0
7UN7r
7I210538
viable
rAIlway
diVe
oppoNEnt
echo
mEN
COCktaIl
SOil
bLaZE
tag
COnsult
able
COMMENTAToR
GalLon
AnY
EXiT
divINe
HOST
BOASt
brEAK-UP
type
cOlUmniSt
DraMA
dizzy
EXHibIT
0731109n8
piZzA
iNsuRaNCe
dessert
GIFT
hOt
ASSISTaNT
cLay
bulLY
kidnap
SORE
magnetic
KinDeRgArTEn
SWING
PleDGE
hONOR
advertising
SmAll
neT
cItizEN
821I75
passagE
ALLOT
iDenTIty
sew
CONViCT
MeANinG
BAnqUeT
deaL
circulate
DISORDer
vIoleNCE
BACON
OFFSPrING
queuE
digiTal
jeT
Complaint
government
hUrricAne
edge
spit
demonstrate
poSe
HeiGht
receipt
RazoR
illegAl
DRAfT
ErOsIoN
24I6WA3
t0i0
LIAr
8ONFES9I82
MILLiON
sENTEnCE
DiGNITY
enGINeeRiNG
peaCh
humorOusly
RiPE
cAndidaTe
casHIeR
gUESS
crYstal
cash
AdMiRaL
Handsome
FLeE
HARD
WHaLe
eqUaLity
integRitY
tRaDitioN
trANQuiL
RainfAlL
maGiCian
PURSUIT
t95r626
rude
CASUAL
CENter
8913e0
587E43231
opTiCaL
democraT
afterwarD
INtERrUpTIoN
ImPORTANCe
TAME
bias
standard
6NG2y
parTiculaR
kIll
POLISH
pOny
GRIm
8vID13C2
CIGaR
cHurch
COMIC
dESPAIrING
frosT
coMMutER
sportsman
TranQuiLIzER
WEst
siMPlify
cEnTIGrADe
IDeOLOGY
poPUlar
loan
membership
Seat
succESSfUl
APPROVAL
journALiSt
gOLd
RENOWNEd
RID
SEVERELy
buffet
sweAT
ROBIN
TRANSCEND
Clown
THIRsTY
certain
HOusEkeeper
pOstUrE
tIDe
Peep
LEgAlLy
GReeN
ANtIBIotic
sTIMULuS
BReed
AgING
HYSTEriCaL
ReGioN
GorGeOuS
EAST
hurt
AIM
ERR
ADMINIStRAtIVE
S1C812A46
ImITATIoN
EXpLIcit
iNsert
research
ACcouNT
mIsTreSs
ARCH
POLICEMAN
aDapt
sAVAGE
MARKET
AltERnATIvelY
shampoO
rANsOm
RED
cOpe
laWsuit
ToPIC
song
sl21per
SHELL
COOK
STaRT
drIll
PLOW
suPerfLuous
NOTe
