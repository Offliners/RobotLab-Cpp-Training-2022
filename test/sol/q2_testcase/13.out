n. 討厭的事物、人、行為等
v. 趕走
n. 橫切
n. 孿生子之一
n./v. 自豪
n. 控制；熟練
n. 腫瘤
n./v. 群；團體；聚集
n. 賓果遊戲
n. 突擊
n. 鞋帶；花邊
adj. 纖弱的；雅緻的
n. 家庭教師
adj./n. 十九
n. 任務
n. 牙膏
adj. 傑出的
v. 捐贈
adj. 基本的
n. 立場
n. 教授
n. 蜻蜓
n. 溫室
adj. 省的
v./n. 落下
prep. 像；和·一樣
adj./n. 空白的
adj./v. 滿足的
v. 想
n. 毫克
n. 現代化
n./v. 預測；預言
n./v. 快照；突然折斷
adj. 優雅的
n./v. 雷聲；打雷
n. 廟宇
n./v. 舞蹈
n. 願望
v. 愛撫
adj. 很可能的
n. 前言
n. 謎題
n. 螞蟻
n. 網
n. 房間；室
n. 大道
Unknown
n. 穩定性
n. 浴缸
adj./n. 邪惡的
n. 海灣；深淵
v. 使能夠
n. 洗禮
n. 領巾
v. 反抗
n. 馬夫
Unknown
v. 期望
adj. 廣大的
n. 設計師
n. 陽光
n./v. 架構
v. 使悲傷
pron. 我們
n. 善良
n. 居民
n. 廢物；垃圾
n. 四分之一
n. 羊
n. [pl. ]腳
adj. 實際的
n. 胡蘿蔔
Unknown
adj./n. 八十
n. 馬鞍
n. 附件
n./v. 群；聚集
adj. 經濟的；節儉的
n. 支撐物
n. 使節團；使命
Unknown
n. 狂熱的崇拜
n. 裙子
n. 反抗者
adj. 正常地；普通地
n. 反抗
n. 貪婪
n./v. 拓荒者；開拓
n. 掌聲
n. 秘書
v. (使)變寬
n. 總數
n. 輪椅
n. 政策
v. 教導
n. 勇敢；勇氣
adj. 笨拙的
adv. 舒適地
n. 不利之條件
n./v. 旅行
n./v. 恐慌
v. 發癢
adj. 不可預測的
n. 手冊
v. 屈辱
Unknown
n. 群
Unknown
n. 障礙；缺點
n. 星期三
n./v.  呼叫；打電話
n. 洗衣店
v. 飄浮；搖擺
n. 岸；濱
adj. 有禮貌的
n. 虛偽
n. 鳳梨
v. 表現；行為舉止
n. 教導；指示
n. 公園
n. 廁所
n. 相似
v. 變成
Unknown
adj./v./n. 破產的
prep. 每
n./v. 等候
n. 粉末
n. 資料
n. 付費高速公路
n. 悲痛
v./n. 捕獲；獲物
n. 政治集團
adj. 感到羞恥的
n. 口譯者
v. 問候
n. 失敗
v. 關聯
n. 部長；外交使節
n. 狀態
adj. 空白處的；邊際的
adj./n. 共和國的；共和主義者
v. 使多樣化
adj./v. 完全的；完成
adj. 過時的
n. 髮型
adj. 必死的
adj. 看不見的
adj./n. 相反的
n. 少女
n. 缺口；峽谷；隔閡
n. 行人
v. 做；到達；獲得
n./v. 球棒
n. 市長；鎮長
n. 不屈不撓
n. 惡夢
n. 電話
adj./n. 女的；雌性的
Unknown
Unknown
n. 報告者
n. 義務；稅
adj. 急躁的
n./v. 解放
adj. 好奇的
n. 變換
n. 錯誤；過失
n. 生命
v. 使發出聲
Unknown
adj. 有熱忱的
Unknown
adj. 操作的；管理的
n. 綽號
n. 差別
n. 光線
n. 審判
v./n. 劃分；分歧
n. 單位
n. 偽君子
n. 祖國；家夠卜
adv. 其他；另外
adj. 金色的；貴重的
n. 達到
n. 元首
adj. 白尊的
n. 儲存
n. 青少年
n. 公園
n. 記錄
v. 遵守
n. 服務員
Unknown
v. 歌唱
n. 假設
adv. 謙虛地
n. 巧合
v.(n. )給～提供娛樂
adj. 天真無邪的
n. 自傳
n. 烤箱
n. 倫理
n./v. 金銀財寶；珍惜
n. 南極或北極
v. 典當
n./v. 停留
v. 攪動
adj. 善於交際地
n./v. 法官
adj. 不重要的
n./v. 偶然；機會
adj. 甜的
n. 姪兒；外甥
v. 畫；拖拉
n. 忠誠
adj. 看得見的
adj. 對…不感興趣
n. 理髮師
n. 馬鈴薯
n. 代理人；仲介人
n. 大水罐；投手
v. 賺取；贏得
n. 適應；改編(本)
v. 使複雜化
n. 輟學者
n. 樣式
n. 博物館
n. 酒保
v. (n)使驚愕
n. 隆隆聲；景氣
adj. 不須負責任的
v. 忽視
n. 閃電；閃光
v. 殘留
n./v. 滾動
n. 影像；形像
n./adj. 悲哀；憂鬱
n. 鑽石
v. 詳述；排演
n. 所有權
Unknown
n. 禮貌；習俗
v. 扭傷
n. 大學
n. 語言學
n. 民主
n. 季節
v. 屈辱
Unknown
n. 發燒；狂熱
n. 演化(論)
n. 環境；圍繞
v. 蹲伏
n. 新聞播報員
prep./adv./conj./n.  曾經；一旦
n. 牛肉
n. 排水系統
v. 去；從事
n./prop. 北京
adj. 明確的；肯定的
n. 習慣
v. 抓(傷)
v./n. 結束；完成
n. 稻草；麥桿
adj. 盡責的
Unknown
adj./n.  牙齒的；齒音(的)
n. 類型
adj./advJpron./conj.  任何的；無論什麼
n. 灰泥
n./v. 運動；鍛鍊
n. 蟑蚣卜；油蟲
n. 識別；身分證明
v. 發明；創造
n. 搬行李工人
n./v. 跳躍
n. 一分錢
n. 膝部
n. 小報
adj. 光滑的
adj./adv./v. 進一步的
n. 表面
n. 小巷子；小徑
v. 挑選
n. 快樂
adj./n. 九十
n. 溝通；訊息
n. 組成部分
n. 障礙
adj. 相當大的
ad v. 痛苦地
adj. 做為…後果的
n./v. 導致；原因
n. 野蠻人
v. 號叫
adj. 令人驚訝的
Unknown
n. 單調
n. 經濟學
adj./n. 棕色的
v. 選舉
n./v. 打盹
adj. 可交換的
adv. 私下地
n. 試金石；(檢驗)標準
n./v. 爭吵
adj. 電的
n. 磁鐵
n. 一長列
adj. 心胸寬闊的
n. 蜂蜜
n. 公司；同伴
n./v. 閃爍
Unknown
adj. 政府的
adj./pron.  許多
adv. 主要(地)
Unknown
n. 蠟燭
n./v. 罐頭；金屬容器
adv. 有朝一日
adv. 不願意地
n. 國家；鄉下
n. 世代；產生
n. 革新
adj. 聰明的
adj. 擁擠的
n. 假日
v. 拆卸
adj. 戶外的
n./v. 利益；興趣
n. 吸引力
n. 保留
n. 離合器；緊握
adv. 在室內
adv./adj./n. 樓下
n. 探訪者
n. 陽光
adj. 必要的；精華的
adj./n./v. 迷戀
n. 拳擊
v. /n贊助(者)
adj./adv. 接近的
adv. 相對地
n. 標題
adv. 同樣地
n. 義大利文
v. 不斷地挑剔
adv. 幾乎
n. 工程師
adj. 防護的；保衛的
n. 生物化學
n. 責任
adj. 持久的
v. 描繪
adj. 明顯的
n. 活動；活動力
n. 決心
n. 能力；才能
n. 移民
adj. 每個
adv. 謙虛地
adj. 特大號的
adj. 室內的
n./v. 堆；疊
v. 使再循環；使回收使用
n. 消費
n. 樂觀；樂天
Unknown
n./v. 睡眠
n. 菜單；項目單
n. 競爭者
n. 手臂
adj./adv./n. 足夠的
Unknown
Unknown
Unknown
adj. 切實可行的
n. 鐵路
v./n.  跳水；潛水
n. 對手
n. 回聲；回響
n.  man的複數
n. 雞尾酒
n./v. 土壤；弄髒
n. 烈火；光輝
n. 標籤
v. 與…商量
adj. 能；有能力的
n. 評論家
n. 加侖
adj./adv./pron. 任何的；若干
n./v. 出口
adj. 神的；神聖的
n./v. 主人
v. 自吹自擂；誇耀
n. 瓦解
n. 類型
n. 專欄作家
n. 戲劇
adj. 頭暈目眩的
v./n. 展示
Unknown
n. 比薩
n. 保險
n. 甜點；餐後點心
n. 禮品；天賦
adj. 熱的；熱情的
n. 助理
n. 黏土；泥土
n. 恃強凌弱者
v. 綁架
adj. 疼痛的
adj. 磁鐵的；帶磁性的
n. 幼稚園
n./v. 鞦韆；搖擺
v. 發誓；保證
n./v. 榮譽；使…增光
n. 廣告業
adj. 小的
n./v. 網；用網捕
n. 公民；市民
Unknown
n. 通行
v.  分配
n. 身分；同一性
v. 縫
v./n. 證明有罪；囚犯
n. 意義；重要性
n. 宴會
n ./v. 交易；處理
v. 循環；流通
n./v. 使混亂；騷亂
n. 暴力
n. 燻豬肉；培根肉
n. 後代；後裔
n./v. 行列；排隊
adj./n. 數字式的
n. 噴射機
n. 抱怨
n. 政府；政體
n. 颶風；暴風雨
n./v. 邊緣；使鋒利
n./v. 口水；吐
v. 證明；展示；遊行
n./v. 姿勢
n. 高度
n. 收到；收據
n. 舌」鬍刀
adj. 違法的
n./v. 草稿
n. 侵蝕
Unknown
Unknown
n. 說謊的人
Unknown
n. 百萬
n. 判決
n. 尊嚴
n. 工程
n. 桃子
adv. 幽默地
adj. 成熟的
n. 候選人
n. 出納員
v./n. 猜測
adj./n. 水晶的
n./v. 現金
n. 海軍將官
adj. 英俊的
v. 逃走；逃離
adj./adv. 硬的；艱難的
n. 鯨
n. 相等；平等；均等
n. 廉正
n. 傳統
adj. 寧靜的
n. 降雨
n. 魔術師
n. 追趕
Unknown
adj. 不禮貌的
adj. 偶然的；碰巧的
n./v. 使集中；核心
Unknown
Unknown
adj. 視覺的
n. 民主主義者
adv. 以後
n. 阻礙；干擾
n. 重要性
adj. 馴∣服的
n. 成見
adj./n. 標準
Unknown
adj. 獨特的
v. 殺死；使終止
adj. 波蘭的；波蘭人(語)的
n. 小馬
adj. 嚴厲的
Unknown
n. 雪茄
n. 教堂
adj. 滑稽的
adj. 自暴自棄的
n./v. 霜；凍
n. 通勤者
n. 運動員
n. 鎮靜劑
adj./adv./n. 西方的
v. 使單純
adj./n. 攝氏(的)
n. 意識形態
adj. 流行的
n./v. 借出
n. 會員(資格)
n./v. 座位；坐下
adj. 成功的
n. 同意；認可
n. 新聞記者
n./adj. 黃金(色)
adj. 著名的
n./v. 解除；免談
adv. 嚴重地
n. 自助餐；快餐
n./v. 汗水；出汗
n. 知更鳥
v. 超越
n. 小丑
adj. 口渴的
adj. 確信的
n. 女管家
n. 姿勢
n. 潮汐
v./n. 窺視；偷看
adv. 法律上
adj./n. 綠色的
n. 抗生素
n. 刺激(物)
n./v. 繁衍；滋生
adj. 變老的
adj. 患歇斯底里症的
n. 地區
adj. 華麗的
adj./adv./n. 東方的
n./v. 受傷
n./v. 目標；瞄準
v. 犯錯
adj. 行政的
Unknown
n. 偽造
adj. 清楚的
v. 插入
n./v. 研究
n. 帳戶；描述
n. 女主人
n. 拱門；拱形
n. 警察
v. 使適應
adj. 兇猛的
n./v. 市場
adv. 二擇一地
n. 洗髮(精)
n. 贖金
adj./n. 紅色的
v. 競爭；妥善處理
n. 訴訟
n. 題目
n. 歌曲
Unknown
n. 外觀；外表
n./v. 廚師；烹煮
n./v. 開始
n./v. 鑽頭；鑽床
v. 耕(地)
adj. 過多的
n./v. 筆記；注意
